templates/contribute/style.md

```diff
@@ -116,157 +98,171 @@ See our [documentation requirements](doc.html) for more suggestions and examples
 
 ### Structuring definitions and theorems
 
-These guidelines hold for declarations starting with `def`, `lemma` and `theorem`. For "theorem statement", also read "type of a definition" and for "proof" also read "definition body".
+These guidelines hold for declarations starting with `def`, `lemma` and `theorem`.
+For "theorem statement", also read "type of a definition" and for "proof" also read
+"definition body".
 
-Use spaces around ":", ":=" or infix operators. Put them before a line break rather
-than at the beginning of the next line.
+Use spaces on both sides of ":", ":=" or infix operators. Put them before a line break
+rather than at the beginning of the next line.
 
-Use two spaces to indent.
+In what follows, "indent" without an explicit indication of the amount means
+"indent by 2 additional spaces".
 
-After stating the theorem, we generally do not indent the first line
-of a proof, so that the proof is "flush left" in the file.
+After stating the theorem, we indent the lines in the subesequent proof by 2 spaces.
 ```lean
-open nat
-theorem nat_case {P : nat → Prop} (n : nat) (H1 : P 0) (H2 : ∀ m, P (succ m)) : P n :=
-nat.induction_on n H1 (assume m IH, H2 m)
+open Nat
+theorem nat_case {P : Nat → Prop} (n : Nat) (H1 : P 0) (H2 : ∀ m, P (succ m)) : P n :=
+  Nat.recOn n H1 (fun m IH ↦ H2 m)
```

Comment:
Isn't this only for tactic proofs?

Reply:
Isn't what only for tactic proofs? This web page? No, the old version had plenty of term-mode, and most of the term-mode that exists here I adapted directly from that.